### ISSUE TITLE
Only one global node

### ISSUE THREAD

**Issue by Baransu** [Netflix/vizceral#57]
Because our global view represents Erlang cluster, there are situations when we have only one node. With this setup our global view is empty. Is there any way we can show only one node on global view?

**Comment by jrsquared** [comment]
This is definitely a bug; I'll dig into why it's not showing if there's only one node.

However, if you have only one node, do you need the global view?

**Comment by Baransu** [comment]
You're right. The answer is yes and no. Right now we have region view only for one node in cluster so global view shows communication between nodes. Our goal is to give user access to select inspected node in global view in Vizceral, and showing this one node when only one node is available will make thinks consistent and explicit.

**Comment by aSqrd-eSqrd** [comment]
Are you trying to basically start at the region view level?  Because, I'm very interested in being able to do that as well.  Right, now create a completely made-up Internet and Region node just so I can go into the region view which is what I actually want.  So I don't actually need the Global view either.

I'd be interested in some pointers or guidance on the where and what to look for to "skip" the global level.

**Comment by Baransu** [comment]
You just have to setView to `["INTERNET"]` if you want region view for that node

**Comment by Baransu** [comment]
I've found this is present when this one node has no connections. Temporary solution could be creating void connection to none existing node.

**Comment by Baransu** [comment]
Here is line causing this issue: https://github.com/Netflix/vizceral/blob/master/src/base/trafficGraph.js#L666

Question is, how should we treat orphan nodes. For my use case it's necessary to have at least entry orphan node present. This could be good solution overall to add isEntryNode check for that.

I'll make PR and wait for your response @jrsquared

**Comment by Baransu** [comment]
Because it's related to #55 I'm adding this to #59 PR

**Comment by aSqrd-eSqrd** [comment]
@Baransu,

I think we have some terminology mis-match.  I tried the `setView(['INTERNET'])` like you suggested, but came to realize that what I'm calling "Region view", isn't what you're calling "Region view".

To me, there are 3 "zoom" levels
1. Global (where you see the region nodes connected to Internet)
2. Region (where you see all the "services" contained in a region graphed out... ltrTreeLayout)
3. Detail Node (where you double-clicked a "service" and now see just that node and any nodes connected to the double-clicked node).

What do you call the "zoom" levels?

@jrsquared, what about you?  What's the "official" level/view "names"?

**Comment by jrsquared** [comment]
@aSqrd-eSqrd That is what we call the zoom levels internally too, glad to see I'm not the only one!

If you do not have a global view at all, you can just create a graph that only has a region view and it should just work. You don't need to fake a global view and then set the view to the region view. The example has a global view and region views just because that's how we built it and use it internally, but any graphs or subgraphs are possible.

**Comment by Baransu** [comment]
For us : 
1. cluster
2. node (in cluster)
3. process (Erlang VM process)

but that doesn't matter, it's the same as you.

I know I don't have to fake global view but for us it's important to show one node in global view and allow interaction with that node as well.

**Comment by aSqrd-eSqrd** [comment]
@jrsquared,

<<I'm duplicating this entire comment plus necessary context as a new issue (which it is), see issue #60.>>
 
I was really excited to see that a global node isn't required and immediately tried it out.  It is a change I need, but haven't had the guts to take on yet.

It does kind of work, but there is something crazy happens with event attachments or something.  I took my usual test JSON file which consist of one region, an Internet node and a single connection between them at the global, while the region is has about 30+ nodes interconnected.   I deleted the global render chunk so that only the region node and its children remained.

It graphed just like expected (ltrTree), but none of the child nodes can be zoomed into.  In fact, when I lathered the whole thing up with breakpoints I only get a breakpoint on the single click which opens the details panel, but nothing at all for the double-click that should zoom to the detail level.
